sqlglot/tokens.py

```diff
@@ -924,8 +933,8 @@ def _scan_numeric_string(self, string_start):
 
         try:
             self._add(token_type, f"{int(text, base)}")
-        except ValueError:
```

Comment:
don't change this

Reply:
do something different is base is None and change the method name

Reply:
changing name in just a sec

Reply:
done